lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -521,7 +535,8 @@ def build_image(self):
             pull=self.pull,
             forcerm=self.rm,
             dockerfile=self.dockerfile,
-            decode=True
+            decode=True,
+            cache_from=self.cache_from
```

Comment:
The `cache_from` parameter was added in docker-py 2.1.0, while this module only requires docker-py 1.8.0. Either raise the minimum required version (in documentation and in the client params in `main()`, or, after https://github.com/ansible/ansible/pull/49707 is merged in, take advantage of the `option_minimal_versions` client param.

Reply:
I would avoid raising the minimum if possible. Shouldn't be hard to avoid this here.

Reply:
Done

Reply:
Not completely: `cache_from` is still in `params`, which will be fed into `self.client.build()` and will cause it to die when docker-py is too old.

The best solution is probably to not put `cache_from` into `params` here, but below insert
```
if self.cache_from:
    params['cache_from'] = self.cache_from
```
(Please keep the trailing comma after `decode=True`, though :) )

Reply:
Yes. you are right. I have update it now. https://github.com/ansible/ansible/pull/49787/commits/683102db74f418c14441951efb0dd313b4d665ae Thanks @felixfontein for noticing.